darwin/future/meta/objects/stage.py

```diff
@@ -50,4 +50,23 @@ def move_attached_files_to_stage(self, new_stage_id: UUID) -> Stage:
 
     @property
     def id(self) -> UUID:
+        """Stage ID."""
         return self._element.id
+
+    @property
+    def name(self) -> str:
+        """Stage name."""
+        return self._element.name
+
+    @property
+    def type(self) -> str:
+        """Stage type."""
+        return self._element.type.value
+
+    @property
+    def edges(self) -> List[List[UUID]]:
+        """Edge ID, source stage ID, target stage ID."""
+        edges = []
```

Comment:
Not obligatory, but you could do this with a list comprehension, which are a bit faster, and worth getting used to.
```python
    return [
        [edge.id, edge.source_stage_id, edge.target_stage_id]
        for edge in in self._element.edges
    ]
```

Optional, but consider it, and also have a think about listcomps in general.